### PR TITLE
JavaScript Data Models

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-05-17 at 11 22 12 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Louisa! There is **_a lot_** more data in the screenshot that you haven't accounted for yet. Make sure you are creating a structure that will provide all the data necessary to create that output image

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  [{
+    name: 'The Art of Sql',
+    type: 'book',
+    author: 'Stephane Faroult',
+    totalCost: 33.99,
+    orderNumber: 113 - 2883177 - 264826,
+    orderPlaced: 'July 3rd'
+
+  }]]
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js]: This array is not being stored in a property 🤔

Comment (codingcodymiller) [reply javascript-data-model/order-history.js]: Were you getting an error when you wrote this in your code editor? Make sure you pay attention to yellow and red squiggly lines, they are there for a reason.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
In addition to my other comments, there is still a significant amount of data you are not accounting for, especially about the order itself.

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    items: [{
+    name: 'The Art of Sql',
+    type: 'book',
+    author: 'Stephane Faroult',
+    totalCost: 33.99,
+    orderNumber: 113 - 2883177 - 264826,
+    orderPlaced: 'July 3rd'
+
+  }]
+}
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js]: This syntax is super wrong, if you opened this in the browser, you likely have parsing errors in your console.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by louiewhitz** [comment]
Screen Shot 2022-05-19 at 8 27 55 PM

File: javascript-data-model/order-history.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  items: [{
+    name: 'The Art of Sql',
+    type: 'book',
+    author: 'Stephane Faroult',
+    totalCost: 33.99,
+    orderNumber: 113 - 2883177 - 264826,
+    orderPlaced: 'July 3rd'
+
+  }]
+}];
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js]: This looks like two separate orders right now, you want one order object for each order which will hold an array of all items in that order.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by louiewhitz** [comment]
Screen Shot 2022-05-22 at 6 36 19 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**